app/upgrades/v1/types.go

```diff
@@ -1,7 +1,7 @@
 package v1
 
 type UpgradeDataString struct {
-	BtcStakingParamStr        string
+	BtcStakingParamsStr       string
```

Comment:
@babylon-devops modified this to `BtcStakingParamStr` =>  `BtcStakingParamsStr` since now it is a slice of parameters